DEVELOPMENT.md

```diff
@@ -21,90 +28,159 @@ npm install
 make
 ```
 
-One liner (with SSH):
+## How to develop a feature
+
+```sh
+# Be sure to be on an up-to-date `develop`
+git checkout develop
+git pull
+
+# If you're on a fork, be sure to pull the `upstream` version
+git pull upstream develop
+
+# Create a new feature branch
+git checkout -b feature/my-new-feature
```

Comment:
I'm not really strict about `feature/` and `hotfix/`, it's indeed Git Flow but it's not a problem IMO if this kind of prefix is not used.